src/include/daos/object.h

```diff
@@ -430,7 +427,7 @@ void daos_iods_free(daos_iod_t *iods, int nr, bool free);
 daos_size_t daos_iods_len(daos_iod_t *iods, int nr);
 
 int daos_obj_generate_oid_by_rf(daos_handle_t poh, uint64_t rf_factor,
-				daos_obj_id_t *oid, daos_ofeat_t ofeats,
+				daos_obj_id_t *oid, daos_otype_t type,
```

Comment:
I am not sure, it might makes sense to have this change, now ofeats are not part of object ID, and instead we used type. And acutally we only support several kinds of types(which was several ofeats Or operations). So actually type is more friendly to users? and less error-prone with disk format changes.

Reply:
not an API change. this is internal